src/cameras/OrthographicCamera.js

```diff
@@ -114,7 +113,7 @@ class OrthographicCamera extends Camera {
 
 	toJSON( meta ) {
 
-		const data = Object3D.prototype.toJSON.call( this, meta );
+		const data = super.toJSON( meta );
```

Comment:
Not sure why this wasn't done in the first place 🤔

Reply:
I think they just missed it.